e2e/ctl_v2_test.go

```diff
@@ -258,7 +258,16 @@ func TestCtlV2Backup(t *testing.T) { // For https://github.com/coreos/etcd/issue
 	epc2 := setupEtcdctlTest(t, &cfg2, false)
 
 	// check if backup went through correctly
-	if err := etcdctlGet(epc2, "foo1", "bar", false); err != nil {
+	var err error
```

Comment:
we should wait until the member published its name, url. then we can do the get, right?